client/components/QuestionInput.js

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react'
+import {Form, Select, Button} from 'semantic-ui-react'
+
+const options = [
```

Comment:
I don't think we are using this variable options anywhere :) I think it can be deleted too.